documentation/tags.md

```diff
@@ -108,7 +108,9 @@ For
 Foreign Key
 Form
 Frameworks
+Function Composition
 Functions
+Functional Programming
```

Comment:
```suggestion
Functions
Functional Programming
```

"Function Composition" is a little too specific for "Functional Programming" is broad enough to be considered a tag so let's add that! 😄